contributing-guides/style-guide.md

```diff
@@ -481,6 +478,19 @@ When documenting optional placeholders like paths or file extensions, it is sugg
 
 - Use `{{path/to/source.ext}}` instead of `{{path/to/source.tar[.gz|.bz2|.xz]}}`.
 
+### Keypress syntax
+
+To mark keypresses for TUI or UI programs, use angle brackets `<` and `>`.
+
+- Single characters example: `<a>`.
+- Special keys are to be written with pascal case: `<Ctrl>`, `<Super>`, `<Alt>`, `<Shift>`, `<Cmd>`, `<Option>`, `<Windows>`, `<Enter>`, `<Home>`, `<Space>`, `<Esc>`, `<ArrowUp>`, `<ArrowLeft>`, `<PageUp>`, `<F5>`, `<F12>`, ...
```

Comment:
Should arrow keys be ArrowUp ArrowDown ArrowLeft ArrowRight or Up Down Left Right?
ArrowDirection is more self evident and just Direction is more condensed.